### ISSUE TITLE
Sleep doesn't work when lid closed - v4.1

### ISSUE THREAD

**Issue by svobluk** [jsassu20/Lenovo-ThinkPad-T450-macOS-Catalina#39]
Hi, I have upgraded from release3.1 to the v4.1 and then upgraded Mojave to the Catalina, everything went quite smooth, hovewer till that time, laptop is not going to sleep when lid is closed. I have to manually go to the menu and click suspend. Any idea where to look? Thx

I'm running T450. 

edit: I have tried set hibernationmode to the 0 via pmset, unfortunately with no change.

**Comment by iceskullpt** [comment]
Weird, I just installed Catalina last night and sleep on close lid works as expected, in fact everything does expect the headphone jack, im on a t450s

**Comment by svobluk** [comment]
would be possible to share pmset -g, thank you

**Comment by iceskullpt** [comment]
![image](https://user-images.githubusercontent.com/34812559/71324165-7f42b900-24d3-11ea-8996-ba1fa135496c.png)

**Comment by svobluk** [comment]
I have some differences, but hibernatemode value is same, I thought it could be different, but in case "3" is working for you, probably it should be working for me as well....anyway thanks a lot, will try to search elsewhere.

**Comment by iceskullpt** [comment]
Good luck! any screenshot tell me and I post here!

**Comment by svobluk** [comment]
I have done TimeMachine/SuperDuper backup and did a clean Catalina installation. It works now!

**Comment by svobluk** [comment]
sleep problem on lid close is back :-/  my assumption that something related to the USB prevent the notebook sleep

Kernel Assertions: 0x4=USB
   id=501  level=255 0x4=USB mod=01.01.1970 1:00 description=com.apple.usb.externaldevice.14800000 owner=Integrated Camera
   id=502  level=255 0x4=USB mod=01.01.1970 1:00 description=com.apple.usb.externaldevice.14700000 owner=Bluetooth USB Host Controller

**Comment by svobluk** [comment]
It seems I have found an issue. Looks like the clover v5101 was the culprit and prevented notebook sleep on lid close. 

It took me several attempts of re-installations and trying different kexts one by one, power management settings etc. The last thing I tried was Clover .... ( Since I did an update from v5097 -which is part of James's release to the v5101 )... and bingo!, reverted back to v5097 and it worked....I have updated now to the v5102 and this Clover version works as well....
